### PR TITLE
CA2009 VB analyzer false negative case fix

### PR DESCRIPTION
**Pull request by TKharaishvili**
Addresses the issue - https://github.com/dotnet/roslyn-analyzers/issues/2283

### CODE REVIEW

**Comment by TKharaishvili** [comment]
@genlu the build was successful. Let's get this thing going.

File: src/Microsoft.NetCore.Analyzers/Core/ImmutableCollections/DoNotCallToImmutableCollectionOnAnImmutableCollectionValue.Fixer.cs [hunk 1]
```diff
@@ -70,7 +70,7 @@ private static Task<Document> RemoveRedundantCall(Document document, SyntaxNode
 
         private static SyntaxNode GetInstance(IInvocationOperation invocationOperation)
         {
-            return invocationOperation.TargetMethod.IsExtensionMethod && invocationOperation.Language != LanguageNames.VisualBasic ?
+            return invocationOperation.TargetMethod.IsExtensionMethod && (invocationOperation.Language != LanguageNames.VisualBasic || invocationOperation.Instance == null) ?
```
Comment (genlu) [inline src/Microsoft.NetCore.Analyzers/Core/ImmutableCollections/DoNotCallToImmutableCollectionOnAnImmutableCollectionValue.Fixer.cs]: Please create a helper for this, and share it between analyzer and fixer.

**Comment by TKharaishvili** [comment]
@genlu the requested changes were made. Round #2 please.

**Comment by genlu** [comment]
Thanks! @TKharaishvili

**Comment by TKharaishvili** [comment]
Thanks to you too @genlu for helping me do this. We might actually make a good team :laughing:

**Comment by TKharaishvili** [comment]
@genlu btw, should I close the issue? Now that the PR is merged in?

**Comment by genlu** [comment]
@TKharaishvili Closed

**Pull request merged**